admin-ui/app/utils/ApiUtils.ts

```diff
@@ -39,6 +39,23 @@ export function handleTypedResponse<T>(
   }
 }
 
+export function handleResponseTyped<T>(
```

Comment:
Two similar functions (`handleTypedResponse` and `handleResponseTyped`) exist with nearly identical logic. Consider consolidating them to reduce duplication.